scripts/lib/api/processHtml.ts

```diff
@@ -39,24 +39,24 @@ export function processHtml(options: {
   const isReleaseNotes = url.endsWith("release_notes.html");
   const images = loadImages($, $main, url, imageDestination, isReleaseNotes);
   if (url.endsWith("release_notes.html")) {
-    setReleaseNotesHeading($, releaseNotesTitle);
+    renameAllH1s($, releaseNotesTitle);
```

Comment:
I think there's still a little bit of use of this being a dedicated function because it keeps the main block extremely _declarative_ rather than _imperative_.